pkg/networkmanager/firewall.jsx

```diff
@@ -175,7 +175,7 @@ function ZoneSection(props) {
             <CardTitle>
                 <Flex alignItems={{ default: 'alignSelfBaseline' }} spaceItems={{ default: 'spaceItemsXl' }}>
                     <Title headingLevel="h2" size="xl">
-                        { cockpit.format(_("$0 Zone"), upperCaseFirstLetter(props.zone.id)) }
+                        { cockpit.format(_("$0 zone"), props.zone.name ? props.zone.name : upperCaseFirstLetter(props.zone.id)) }
```

Comment:
`props.zone.name || upperCaseFirstLetter(props.zone.id)` is the more useful notation for this. Unfortunately JS does not understand `val ?: fallback`, like C.